### PR TITLE
Adding check for queue length using redis

### PR DESCRIPTION
**Pull request by gurvirsingh15**

### CODE REVIEW

File: celery_prometheus_exporter.py [hunk 1]
```diff
@@ -236,8 +238,8 @@ def __init__(self, app, queue_list):
     def measure_queues_length(self):
         for queue in self.queue_list:
             try:
-                length = self.connection.default_channel.queue_declare(queue=queue, passive=True).message_count
-            except (amqp.exceptions.ChannelError,) as e:
+                length = self.redis_client.llen(queue)
```
Comment (GorrieXIV) [inline celery_prometheus_exporter.py:241]: just making sure: should this be `llen` and not `len`?

Comment (GorrieXIV) [reply celery_prometheus_exporter.py:241]: also what about the case where redis_client is not supplied to this class? Do we only use `QueueLengthMonitoringThread` in this situation? =

Comment (gurvirsingh15) [reply celery_prometheus_exporter.py:241]: Yeah it should be llen. As far as I know, only main() instantiates `QueueLengthMonitoringThread` so it should be alright

Comment (GorrieXIV) [reply celery_prometheus_exporter.py:241]: good enough for moi

**Pull request merged**